common/resource/fx.go

```diff
@@ -389,14 +390,14 @@ func ArchiverProviderProvider(cfg *config.Config) provider.ArchiverProvider {
 }
 
 func SdkClientFactoryProvider(cfg *config.Config, tlsConfigProvider encryption.TLSConfigProvider, provider metrics.MetricsHandler) (sdk.ClientFactory, error) {
-	tlsFrontendConfig, err := tlsConfigProvider.GetFrontendClientConfig()
+	tlsClientConfig, err := tlsConfigProvider.GetInternodeClientConfig()
```

Comment:
could someone more familiar with the TLS config check that this makes sense?

Reply:
My understanding the InternodeClient is for history and matching. Worth double check or test this config.